### ISSUE TITLE
Run MATSim for more than one day

### ISSUE THREAD

**Issue by Mayoookh** [matsim-org/matsim-code-examples#287]
Hello, 

For a specific issue (calculating charging demand of electric car users), I would like to run MATSim for more than a day. What would be the easiest way to do that?

Should the population plan and PT plan be kept same as it might get really complex (and impossible?) to provide a week-long plan for both? But keeping them same for over a week, is it possible to run MATSim simulation over a week?

Thank you in advance for a feedback

**Comment by JWJoubert** [comment]
Great questions, @Mayoookh. The short answer is that a) no, MATSim (out of the box) does not run multi-day models (yet); and b) it is something we are toying with, but not at the stage where we can share code (we use it for freight, which also runs over multiple days).

A plausible work-around would be to generate "weekly plans" for each agent. There are many subtle challenges: you suddenly HAVE to work with facility opening times so that the people only go to work during normal day-times. So, on day two, for example, the start time would be `Time.parseTime("24:00:00") + Time.parseTime("06:30:00") = 109800 s` if the work starts at 06:30 am. How the scoring is affected... I'm not quite sure yet.

**Comment by JWJoubert** [comment]
> is it possible to run MATSim simulation over a week?

Yes, the machinery allows for it. Whether it is doing something useful is probably more up to **_you_**. And, of course, the computational burden would be quite extensive.

**Comment by Mayoookh** [comment]
@JWJoubert thank you for your quick and helpful reply. As I expected, running MATSim over a day comes with expected ambiguity. The quickest way out, for now, will be to run it for a day and assume similar behavior of people over the weekdays to perform analysis
